interfaces/Portal/src/app/program/registration-activity-detail-accordion/registration-activity-detail-accordion.component.ts

```diff
@@ -92,7 +92,9 @@ export class RegistrationActivityDetailAccordionComponent implements OnInit {
     );
 
     if (
-      !PaymentUtils.hasVoucherSupport(paymentRow.fsp) &&
+      !PaymentUtils.hasVoucherSupport(
+        paymentRow.programFinancialServiceProviderConfigurationTranslatedLabel,
```

Comment:
Shouldn't this be the "non-translatable", enum-name?

So, `paymentRow.programFinancialServiceProviderConfigurationName`? or `financialServiceProviderName`?